README.md

```diff
@@ -30,6 +30,24 @@ $ npm run start:hmr
 $ npm run start:prod
 ```
 
+## Database migration workflow
+Automatic migration generation creates a new migration file and writes all sql queries that must be executed to make a new database or to update the database.
+```bash
+$ node_modules/.bin/ts-node ./node_modules/.bin/typeorm migration:generate -n 'name-of-choice'
```

Comment:
'name-of-choice' is the name of your new migration file? Maybe clarify that a bit more.